tests/rt.conf

```diff
@@ -204,122 +204,233 @@ RUN     | hrrr_control_decomp_dyn32_phy32
+RUN     | hrrr_control_restart_dyn32_phy32                                                                                        | + cheyenne hera |     | hrrr_control_dyn32_phy32
+
+COMPILE | 43 | gnu | -DAPP=ATM -DCCPP_32BIT=ON                                                                                               | + cheyenne hera | fv3 |
+RUN     | rap_control_dyn64_phy32                                                                                                 | + cheyenne hera | fv3 |
+
+COMPILE | 44 | gnu | -DAPP=ATM -D32BIT=ON -DCCPP_32BIT=ON -DDEBUG=ON                                                                         | + cheyenne hera | fv3 |
+RUN     | rap_control_debug_dyn32_phy32                                                                                           | + cheyenne hera | fv3 |
+RUN     | hrrr_control_debug_dyn32_phy32                                                                                          | + cheyenne hera | fv3 |
+
+COMPILE | 45 | gnu | -DAPP=ATM -DCCPP_32BIT=ON -DDEBUG=ON                                                                                    | + cheyenne hera | fv3 |
+RUN     | rap_control_dyn64_phy32_debug                                                                                           | + cheyenne hera | fv3 |
+
+##################################################################################################################################################################
+# S2S tests                                                                                                                                                      #
+##################################################################################################################################################################
+
+COMPILE | 46 | gnu | -DAPP=S2SWA -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp                                   | + cheyenne hera | fv3         |
+RUN     | cpld_control_p8                                                                                                          | + cheyenne hera | fv3         |
+
+COMPILE | 47 | gnu | -DAPP=S2S -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp                                     | + cheyenne hera | fv3         |
+RUN     | cpld_control_nowave_noaero_p8                                                                                            | + cheyenne hera | fv3         |
+
+COMPILE | 48 | gnu | -DAPP=S2SWA -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp                        | + cheyenne hera | fv3         |
+RUN     | cpld_debug_p8                                                                                                            | + cheyenne hera | fv3         |
+
```

Comment:
You are missing the new unstruc WW3 tests here.

Reply:
Thanks. I was afraid of that already, so since I know the PRs before this one do not change rt.conf i'm currently using develop's rt.conf and rt_gnu.conf to create an rt.conf for this PR, which should address this.

Reply:
Resolving with latest commits!